tools/perf/perf_run.py

```diff
@@ -578,23 +578,24 @@ def recordStats(backend, timings, precision, batch_size=1, compile_time_s=None):
             if model_torch is not None:
                 model_torch = model_torch.half()
 
-        status = run(
-            model,
-            backends,
-            input_tensors,
-            params,
-            precision,
-            batch_size,
-            is_trt_engine,
-            model_torch=model_torch,
-        )
+        with torch.no_grad():
+            status = run(
+                model,
+                backends,
+                input_tensors,
+                params,
+                precision,
+                batch_size,
+                is_trt_engine,
+                model_torch=model_torch,
+            )
 
     # Generate report
     print("Model Summary: ", model_name)
     summary = pd.DataFrame(results)
+    summary["model_name"] = (
```

Comment:
a minor comment - Does model_name get appended as the last column here? 
Instead can we have 
model_name, Batch size, Precision .....  ?